testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml

```diff
@@ -67,7 +67,7 @@
     </extension>
 
     <extension qualifier="webdriver-javascriptbrowser">
-        <property name="browser">${js.browser}</property>
+        <property name="browser">phantomjs</property>
```

Comment:
Why to hardcode phantomjs as the JS browser when there's a property for that? We sometimes need to use e.g. Chrome for debugging purposes.

Reply:
Fixed. Thanks!